src/test/java/org/fcrepo/client/TransactionBuilderTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree.
+ */
+package org.fcrepo.client;
+
+import java.net.URI;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+/**
+ * Tests to check the TransactionBuilder constructor works as expected. Method testing done in TransactionBuilderIT.
+ *
+ * @author mikejritter
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class TransactionBuilderTest {
+
+    private static final String SERVER_ENDPOINT = "http://localhost:8080";
+
+    @Mock
+    private FcrepoClient client;
+
+    @Test
+    public void testStartValidUris() {
```

Comment:
Personally I would split this up into 4 tests, since there isn't any overlap between the cases